Libraries/Azure.psm1

```diff
@@ -2495,6 +2495,19 @@ Function Set-SRIOVinAzureVMs {
 			$AllNics = Get-AzNetworkInterface -ResourceGroupName $ResourceGroup `
 			| Where-Object { $($_.VirtualMachine.Id | Split-Path -leaf) -eq $VMName }
 
+			# Sometimes Get-AzNetworkInterface returns null, retry 5 times
+			$retry_count = 0
+			While ($null -eq $AllNics -and $retry_count -lt 5) {
```

Comment:
try more times, like 120. It means two minutes.

Reply:
Done